tests/functional/prometheous/test_prometheus_validations_file.py

```diff
@@ -333,3 +330,70 @@ def test_prometheus_pv_resize(self):
             pod_name=pod_name, pvc_name=pvc_name,
             filename="secondfilename", dirname="seconddirname",
             metric_data=resize_metrics, operation="create")
+
+    @pytest.mark.tier2
+    def test_prometheus_volume_metrics_on_node_reboot(self):
+        """Validate volume metrics using prometheus before and after node
+        reboot"""
+
+        # Create PVC and wait for it to be in 'Bound' state
+        pvc_name = self.create_and_wait_for_pvc()
+        pod_name = openshift_ops.oc_create_tiny_pod_with_volume(
```

Comment:
cirros pods won't work?

Reply:
I tried cirros pods where dc is created, the test is failing when created with dc, so using tiny pod which is giving the expected results.